tools/sof-dump-status.py

```diff
@@ -452,7 +519,11 @@ def dump_dapm(dapm, filter = "all"):
         exit(0)
 
     if ret_args['power'] is True:
-        sysinfo.loadPower()
+        try:
+            sysinfo.loadPower()
+        except (IOError, NoMapException) as e:
+            print(e, file=sys.stderr)
+            exit(1)
```

Comment:
Don't hide stack traces. This a test tool, stack traces are useful. Just let it fail. This is not C either, no need to implement error handling manually. Exceptions come for free. The less code the better.